src/es/components/atoms/heading/Heading.js

```diff
@@ -188,7 +188,21 @@ export default class Heading extends Shadow() {
         height: var(--h-border-top-height, var(--mdx-sys-sizing-fix-2xs));
         width: var(--h-border-top-width, var(--mdx-sys-sizing-fix-3xl));
       }
-      
+
+      :host([border-top][brand=ibaw]) [border-top]::before {
+        content: unset;
+      }
+
+      :host([border-top][brand=ibaw]) [border-top]::after {
```

Comment:
@Weedshaker Hier gabs bereits das attribut "border-top". Bei IBAW ist's eigentlich kein border sondern "//" am Ende der Zeile. Man könnte es auch umbenennen in "decorated" aber weiß nicht obs das wert ist, weils im BE ja glaube ich schon teils benutzt wird